.github/workflows/pypi.yml

```diff
@@ -18,15 +18,15 @@ jobs:
       name: release
     steps:
       - uses: actions/checkout@v4
-      - name: Set up Python
+      - name: py3.10
```

Comment:
These are the step names. I would undo this and only focus on the job names otherwise the step name(s) would be unclear. For example:

<img width="885" alt="Screenshot 2024-04-01 at 1 19 48 PM" src="https://github.com/aws-solutions-library-samples/osml-imagery-toolkit/assets/4109909/fef86fb3-f2b0-460f-966a-7bdce680e7bc">

Reply:
I would prefer as `Set up Python`, `Install dependencies`, etc.